drivers/hmis/app/models/hmis/hud/client.rb

```diff
@@ -43,6 +43,8 @@ class Hmis::Hud::Client < Hmis::Hud::Base
   has_many :current_living_situations, through: :enrollments
   has_many :hmis_services, through: :enrollments # All services (HUD and Custom)
   has_many :custom_data_elements, as: :owner
+  has_many :client_projects
+  has_many :projects_including_wip, through: :client_projects, source: :project
```

Comment:
I think we just want `has_many :projects, through: :client_projects, source: :project`. Is that doable?

Reply:
Unfortunately that change does cause test failures. Maybe we could roll this into another issue? Also I just noticed that projects_including_wip isn't actually needed anymore